### PR TITLE
Onboarding-ecoinvent-new-version_v10

### PR DESCRIPTION
**Pull request by ysherstyuk**
Hi Sven,

All ecoinvent data from v10 are onboarded, including the remaining tables from database_overview, cut_off_cumulative_LCIA_v3.10 to both raw layer and where relevant to datamodel layer. 

The generation of all tables is tested and all of them are available in the Catalog.

Only input_data is not onboaded because we don't have it yet. However it is not a priority for now, we will add it in the upcoming weeks once the data is available.

### CODE REVIEW

**Review by SKruthoff — CHANGES_REQUESTED** [review]
Hi, see my comments. In general it looks good, just double checking two things. The HS observation in cut-off overview is based on the finding from Sohail, so maybe good to check the other AOs if they also have that column and not just the APSOS ao.

File: functions/datamodel_schema.py [hunk 1]
```diff
@@ -225,6 +225,8 @@
             StructField('product_uuid', StringType(), False),
             StructField('reference_product_name', StringType(), True),
             StructField('unit', StringType(), True),
+            StructField('CPC_code', StringType(), True),
+            StructField('CPC_name', StringType(), True),
```
[the remark this answers is not in the document]
Comment (ysherstyuk) [reply functions/datamodel_schema.py:229]: These are lowercase in the Catalog. Do I need to change it here to lowercase then?

File: functions/landingzone_schema.py [hunk 2]
```diff
@@ -339,7 +341,7 @@
         ]
```
[the remark this answers is not in the document]
Comment (ysherstyuk) [reply functions/landingzone_schema.py:341]: Hm I think I was adding everything based on the new setup of the data. Can you elaborate what you mean by HS Classification?

File: functions/datamodel_schema.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ysherstyuk) [reply functions/datamodel_schema.py:229]: Done! Changed to lowercase

File: functions/landingzone_schema.py [hunk 3]
```diff
@@ -360,6 +363,7 @@
             StructField('Product UUID', StringType(), True),
             StructField('Reference Product Name', StringType(), True),
             StructField('CPC Classification', StringType(), True),
+            StructField('HS2017_Classification', StringType(), True),
```
Comment (SKruthoff) [inline functions/landingzone_schema.py:366]: The HS Classification does not have an underscore in their name in the landingzone. In the raw schema it should have it though.

**Review by SKruthoff — CHANGES_REQUESTED** [review]
There is only one minor change that still needs to be made. Feel free to let someone else (like Alex) approve it once that change is made, to cross off this branch.

File: functions/landingzone_schema.py (hunk 3 above)
Comment (ysherstyuk) [reply functions/landingzone_schema.py:366]: Good point!

**Review by SKruthoff — APPROVED** [review]

**Pull request merged**